src/main/java/com/exasol/adapter/dynamodb/documentnode/dynamodb/DynamodbList.java

```diff
@@ -37,4 +38,8 @@ public DynamodbList(final List<AttributeValue> value) {
     public void accept(final DynamodbNodeVisitor visitor) {
         visitor.visit(this);
     }
+
+    List<AttributeValue> getValue() {
```

Comment:
```suggestion
    public List<AttributeValue> getValue() {
```

Reply:
I am guessing it should be `public`? 
It can be confusing with default modifiers in Java: by default in class they are package private and public in interfaces.

Reply:
I am adding public modifier to the ones in other places, please feel free to change them otherwise.

Reply:
In this case it is inteded to be packet private. The method public interface for this conversion is using the`DynamodbNodeToAttributeValueConverter`